do-arm-configure-sysroot

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+ARCH=arm-none-eabi
+DIR=`dirname $0`
+meson "$DIR" \
```

Comment:
Did you mean to add the -Dsysroot=true option here?

Reply:
Yes! I fix it!